Tools/autotest/sim_vehicle.py

```diff
@@ -1232,6 +1236,10 @@ def generate_frame_help():
 if cmd_opts.strace and cmd_opts.callgrind:
     print("callgrind and strace almost certainly not a good idea")
 
+if (cmd_opts.mcast or cmd_opts.sysid) and cmd_opts.auto_sysid:
```

Comment:
Why can't we use mcast with auto_sysid?

Setting the ardupilot system ID should be orthogonal to binding extra ports etc etc.

Reply:
When I start mavproxy like this:

   sim_vehicle.py -v ArduPlane -n3 --auto-sysid --mcast

It doesn't show multiple links. When I start it like this:

   sim_vehicle.py -v ArduPlane -n3 --auto-sysid 
   
It does. 

MAVProxy not displaying the mcast: link may be a bug. I even added the link manually via the command line with `link add mcast:` and it doesn't show up in the console window. In all cases, however, I can see the link by issuing `link list` on the command line.

I can take mcast out of that conditional if you want but I wanted to make the group aware of this possible bug.

Reply:
update: if I switch to each vehicle with a `vehicle x` command then I am able to have mcast links show up. I have to issue them for each vehicle individually though.

Reply:
Works after a rebase against master! Who'd a thunk it!